### ISSUE TITLE
MissingPropertyException: No such property: antBuilderInput

### ISSUE THREAD

**Issue by dwittner** [Liverpool-Victoria/gradle-weblogic-plugin#8]
I get the following Exception:

```java
Caused by: groovy.lang.MissingPropertyException: No such property: antBuilderInput for class: com.lv.plugins.weblogic.tasks.WLDeployTask_Decorated
	at org.gradle.api.internal.plugins.ExtraPropertiesDynamicObjectAdapter.setProperty(ExtraPropertiesDynamicObjectAdapter.java:46)
	at org.gradle.api.internal.CompositeDynamicObject.setProperty(CompositeDynamicObject.java:122)
	at com.lv.plugins.weblogic.tasks.WLDeployTask_Decorated.setProperty(Unknown Source)
	at com.lv.plugins.weblogic.tasks.AbstractWLDeployTask.getAntBuilderInput(AbstractWLDeployTask.groovy:152)
	at com.lv.plugins.weblogic.tasks.AbstractWLDeployTask.run(AbstractWLDeployTask.groovy:115)
```

What am i doing wrong?

**Comment by willis7** [comment]
Can you share your build file and what version of Gradle you are using?

**Comment by dwittner** [comment]
```
apply plugin: 'java'
apply plugin: 'eclipse'

task wrapper(type: Wrapper) {
    gradleVersion = '2.4'
}

buildscript {
	repositories {
	    mavenCentral()
	    maven { url 'https://dl.bintray.com/lv/gradle-plugins/' }
	}
	dependencies {
        classpath 'com.lv.plugins:gradle-weblogic-plugin:0.3'
    }
}

apply plugin: 'com.lv.weblogic'

sourceCompatibility = 1.5
version = '1.0'
jar {
    manifest {
        attributes 'Implementation-Title': 'Gradle Quickstart',
                   'Implementation-Version': version
    }
}

repositories {
    mavenCentral()
}

weblogic {
    adminurl = 't3://localhost:7001'
    deploymentName = 'myUberApp'
    source = "${projectDir}/dist/demo.war"
    targets = 'AdminServer'
    user = 'weblogic'
    password = 'secret'
    debug = false
    verbose = false
    upload = true
    remote = false
}

dependencies {
	weblogic files('C:/Oracle/Middleware/Oracle_Home/wlserver/server/lib/wlfullclient.jar')
    compile group: 'commons-collections', name: 'commons-collections', version: '3.2'
    compile 'javax:javaee-api:7.0'
    testCompile group: 'junit', name: 'junit', version: '4.+'
}
```

**Comment by willis7** [comment]
Please try the latest version of the plugin found via the link below.

(Plugin Portal)[https://plugins.gradle.org/plugin/com.lv.weblogic]

**Comment by dwittner** [comment]
The problem is fixed for me with version 0.4. Thanks a lot. Good Job!
